addons/web/static/src/views/relational_model.js

```diff
@@ -574,11 +574,15 @@ export class Record extends DataPoint {
         this.invalidateCache();
     }
 
+    async _askLocalChanges() {
```

Comment:
in the BasicRelationalModel, there's a similar function but called `askChanges`. So I would keep the same name.